src/definition/accessors/IModerationModify.ts

```diff
@@ -7,4 +10,39 @@ export interface IModerationModify {
      * @param appId the app id
      */
     report(messageId: string, description: string, userId: string, appId: string): Promise<void>;
+
+    /**
+     * Provides a way for Apps to delete a message.
+     * @param message the message to be deleted
+     * @param user the user who is deleting the message
+     * @param reason the reason for deleting the message
+     * @param action the action to be taken
+     * @param appId the app id
+     * @returns
+     * @throws {Error} if the message is not found
+     * @throws {Error} if the user is not found
+     */
+    deleteMessage(message: IMessage, user: IUser, reason: string, action: string, appId: string): Promise<void>;
```

Comment:
Since this method is modifying messages, it should be in the Messages bridge

Reply:
Thank you! Resolved.